ansible/files/queries.yml

```diff
@@ -0,0 +1,30 @@
+pg_database:
+  query: "SELECT pg_database.datname, pg_database_size(pg_database.datname) as size_bytes FROM pg_database"
+  master: true
+  cache_seconds: 30
+  metrics:
+    - datname:
+        usage: "LABEL"
+        description: "Name of the database"
+    - size_bytes:
+        usage: "GAUGE"
+        description: "Disk space used by the database"
+
+pg_stat_database:
+  query: "SELECT sum(numbackends) as numbackends FROM pg_stat_database"
+  master: true
+  metrics:
+    - numbackends:
+        usage: "GAUGE"
+        description: "The number of active backends"
+
+pg_stat_statements:
```

Comment:
This metric is probably the weirdest one here, in that it might be misleading unless you're paying attention to what the data means. Open to suggestions for better ones!

Reply:
This also requires `pg_stat_statements` to be enabled on the database (we should probably do this by default), so we should check that it doesn't fail drastically when it is disabled